File: 백준/Bronze/10807. 개수 세기/개수 세기.cc
#include <stdio.h>

int main(void) {
	int arr[100];
	int n, i, v, cnt=0;
	scanf("%d", &n);
	for(i=0; i<n; i++) {
		scanf("%d ", &arr[i]);
	}
	scanf("%d", &v);
	for(i=0; i<n; i++) {
		if(arr[i]==v)
			cnt++;
	}
	printf("%d", cnt);
}

File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner num = new Scanner(System.in);
        int[] arr = new int[100];
        int cnt = 0;

        int n = num.nextInt();
        for (int i = 0; i < n; i++) {
            arr[i] = num.nextInt();
        }
        int v = num.nextInt();
        for (int i = 0; i < n; i++) {
            if(arr[i]==v)
                cnt++;
        }
        System.out.println(cnt);
    }
}